site/public/js/ta-grading.js

```diff
@@ -592,8 +592,8 @@ function updateCookies(){
 // Student navigation
 function gotoMainPage() {
 
-  let window_location = $("#main-page")[0].dataset.href
-
+  let window_location = $("#main-page")[0].dataset.href;
+  
```

Comment:
This change seems unrelated to the rest of this PR so it would probably be best to revert it to prevent accidental merge conflicts.